ansible/slurm-drain.yml

```diff
@@ -0,0 +1,38 @@
+---
+- import_playbook: cluster-infra-query.yml
```

Comment:
Is it worthwhile making a separate playbook for cluster-infra-query?  Might be equally succinct to invoke `stackhpc.cluster-infra` directly here and save a layer.

Reply:
Makes sense